trustymail/domain.py

```diff
@@ -151,6 +151,6 @@ def generate_results(self):
     def format_list(self, record_list):
 
         if not record_list:
```

Comment:
Given the role this function is playing, and how ambiguous `if not var` can be, it might be helpful to identify which values you're trying to check here. Is this meant to catch `[]`, `False`, `None`, `""` (which I believe are all considered false-y by Python) or some combination of them? If it's deliberately meant to catch multiple possibilities, then this makes the most sense, though I might add a comment saying what values might get collapsed into a null output in this function.

Reply:
Only lists should be passed to the function, so it's really only meant to catch `[]`.  I'll make the change.

Reply:
Yeah, I wouldn't recommend `if len(record_list)`, I more meant making it explicit that this is how you handle empty lists, not `None` or other values. You could also do that in a comment, or you could do `== []`. A comment might suffice here.